README.md

```diff
@@ -118,6 +118,14 @@ Getting the currently set value for the option would then be done with:
 surfactant config core.recorded_institution
 ```
 
+### Configuration Options
+
+- **docker.enable_docker_scout**: Controls whether Docker Scout is enabled. Default is `true`. To disable Docker Scout, run:
+
+```bash
+surfactant config docker.enable_docker_scout false
+```
+
```

Comment:
Can you add an entry for it to https://github.com/LLNL/Surfactant/blob/main/docs/settings.md?

I don't think we need an extensive list of options in the README since we link to the settings page, but since this might be a common setting users want to disable -- maybe we can remove the heading (so it is part of the "Command Line" section and reword it as being "another example of a setting to control information gathered is..."